lib/parallel_tests/cucumber/scenarios.rb

```diff
@@ -12,18 +13,90 @@ def all(files, options={})
           tags = []
           tags.concat options[:ignore_tag_pattern].to_s.split(/\s*,\s*/).map {|tag| "~#{tag}" }
           tags.concat options[:test_options].to_s.scan(/(?:-t|--tags) (~?@[\w,~@]+)/).flatten
+
           split_into_scenarios files, tags.uniq
         end
 
         private
 
+        # Class stolen from cucumber as it was private
+        # Private class can be found here : https://github.com/cucumber/cucumber-ruby/blob/master/lib/cucumber/runtime.rb#L130
+        class NormalisedEncodingFile
```

Comment:
still confused as to what this means ... why not require runtime and then use Cucumber::Runtime:: NormalisedEncodingFile ?